src/Newton.jl

```diff
@@ -125,13 +125,13 @@ function newton(Fhandle, Jhandle, x0, p0, options::NewtonPar; normN = norm, call
 			break
 		end
 	end
-	(resHist[end] > tol) && @error("\n--> Newton algorithm failed to converge, residual = $(res[end])")
```

Comment:
zygote not compatible with `@error` as it uses a `try/catch` statement https://github.com/FluxML/Zygote.jl/issues/386

Reply:
Is it OK with `printstyled(color=:red, "\n--> Newton algorithm failed to converge, residual = $(res)")`?

Reply:
yes